### PR TITLE
Flp 177 merge pages

### PR DESCRIPTION
**Pull request by MichaelYoung1981**

### CODE REVIEW

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Shared/DraftApprenticeship.cshtml [hunk 1]
```diff
@@ -34,375 +32,326 @@
...
-            </div>
-        }
-        else
-        {
-            <input type="hidden" asp-for="IsOnFlexiPaymentPilot" value="false" />
-        }
-    }
+@if (Model is EditDraftApprenticeshipViewModel)
+{
+    <h1 class="govuk-heading-xl">Edit personal details</h1>
```
Comment (dinathain) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Shared/DraftApprenticeship.cshtml:37]: In the prototype/mockups I've seen, I think this header is just "Apprentice personal details" (no "add" or "edit")

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Shared/DraftApprenticeship.cshtml [hunk 2]
```diff
@@ -34,375 +32,326 @@
...
                     </div>
                 </div>
             </div>
         </fieldset>
     </div>
+}
+
+@if (Model is EditDraftApprenticeshipViewModel)
+{
+    <h1 class="govuk-heading-xl">Edit training details</h1>
```
Comment (dinathain) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Shared/DraftApprenticeship.cshtml:162]: Same here, but "Apprentice training details"

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Shared/DraftApprenticeship.cshtml [hunk 3]
```diff
@@ -34,375 +32,326 @@
...
+        <dt class="das-definition-list__title">Apprenticeship delivery model</dt>
+        <dd class="das-definition-list__definition govuk-!-margin-bottom-4">@Model.DeliveryModel.ToDescription()</dd>
+    </dl>
+}
 
-    <div class="govuk-form-group  @Html.AddClassIfPropertyInError(x => x.EndDate, "govuk-form-group--error")">
+<div id="start-date-section" class="govuk-form-group  @Html.AddClassIfPropertyInError(x => x.StartDate, "govuk-form-group--error")">
+    <fieldset class="govuk-fieldset" aria-describedby="start-date-hint" role="group">
+        <legend class="govuk-fieldset__legend govuk-fieldset__legend--s">
+            @startDateLabel
```
Comment (dinathain) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Shared/DraftApprenticeship.cshtml:207]: Should both the "planned" and "actual" start date be visible simultaneously here? Again, just going off the mockups here, it looks like the "actual" start date is not visible for non-pilot providers (but "planned" is), and vice-versa for pilot providers.

**Review by dinathain — COMMENTED** [review]
Only comments from me are around header names and visibility of the start date sections. 

I've added notes on the relevant lines, but all are in reference to differences between this and the mockup for this page.

**Review by dinathain — APPROVED** [review]
Latest changes makes sense - I will update the header text in 185 as discussed. Approved.

**Comment by satya-sanikommu** [comment]
This branch is already merged into FLP-178, this PR is no required

**Pull request closed without merging**